src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  const semyResult = { ...state };
+
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
```

Comment:
Move all cases to the object.

Example of usage:
`case ActionType.AddProperties:`

Reply:
I don't really think that i understand correctly, but I hope